lib/src/element/text.cpp

```diff
@@ -542,13 +542,16 @@ namespace cycfi { namespace elements
       if (_select_start == -1)
          return;
 
+      if (!_is_focus) //No caret if not focused
+         return;
+
       auto& canvas = ctx.canvas;
       auto const& theme = get_theme();
       rect caret_bounds;
       bool has_caret = false;
 
       // Handle the case where text is empty
-      if (_is_focus && _text.empty())
```

Comment:
Why is _is_focus removed? The caret should not be drawn when the text element is not the focus.

Reply:
Ignore this. I failed to see the added if statement above.

Reply:
Ah, sorry about that, I forgot to mention that minor rearrangement.